src/Compilers/CSharp/Test/Semantic/Semantics/ForEachTests.cs

```diff
@@ -1900,8 +1900,7 @@ public static void Main(string [] args)
                 Diagnostic(ErrorCode.ERR_IllegalStatement, "i < 5").WithLocation(6, 18),
                 // (6,25): error CS0103: The name 'i' does not exist in the current context
                 //     foreach(int; i < 5; i++)
-                Diagnostic(ErrorCode.ERR_NameNotInContext, "i").WithArguments("i").WithLocation(6, 25)
-                );
+                Diagnostic(ErrorCode.ERR_NameNotInContext, "i").WithArguments("i").WithLocation(6, 25));
```

Comment:
Consider reverting since only change in file. The old format is actually more convenient when diagnostics need to be updated.